selfhosted/calibration.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+import pathlib
+
+from qibocal.protocols.characterization import Operation
+from qibolab import create_platform
+
+MESSAGE_FILE = "message.txt"
+
+parser = argparse.ArgumentParser()
+parser.add_argument("name", type=str, help="Name of the platform.")
+
+
+def generate_message(name, fidelities, time):
+    """Generates message that is added to GitHub comments."""
+    path = pathlib.Path.cwd() / MESSAGE_FILE
+    with open(path, "w") as file:
+        file.write(f"Run on platform `%s` completed in %.2fsec! :atom:" % (name, time))
+        file.write("\n\nReadout assignment fidelities:\n")
+        for qubit, fidelity in fidelities.items():
+            file.write(f"{qubit}: {fidelity}\n")
+
+
+def main(name):
+    """Execute single shot classification routine on the given platform."""
+    routine = Operation.single_shot_classification.value
+    params = routine.parameters_type.load(dict(nshots=5000))
+
+    platform = create_platform(name)
+    qubits = platform.qubits
+
```

Comment:
I believe that on `qw5q_gold` tests are failing because we are missing the `platform.setup()`, `platform.connect()` and similar...
I forgot to include them in the script that I gave you since I was only testing `dummy`

Reply:
Good point, thanks.